e2e/cypress/support/commands.js

```diff
@@ -83,10 +85,161 @@ Cypress.Commands.add('deleteProject', (projectName) => {
     autoEnd: false,
   });
 
-  cy.contains('.project-card', projectName).find('.anticon-delete').click();
+  cy.get(`[data-test-id="project-card-${projectName}"]`).find('.anticon-delete').click();
   log.snapshot('opened-delete-modal');
 
-  cy.get('.delete-project-modal').find('input').type(projectName);
+  cy.get('.data-test-delete-project-modal').find('input').type(projectName);
   cy.contains('Permanently delete project').click();
   log.end();
 });
+
+Cypress.Commands.add('selectProject', (projectName) => {
+  const log = Cypress.log({
+    displayName: 'Selecting project',
+    message: [`🔐 Selecting project named ${projectName}`],
+    autoEnd: false,
+  });
+
+  cy.get(`[data-test-id="project-card-${projectName}"]`).click();
+
+  log.end();
+});
+
+Cypress.Commands.add('launchAnalysis', () => {
+  const log = Cypress.log({
+    displayName: 'Launching analysis',
+    message: ['launch analysis'],
+    autoEnd: false,
+  });
+
+  log.snapshot('launch-analysis');
+  cy.contains('button', 'Launch analysis').click();
+
+  log.snapshot('launch-first-experiment');
+  cy.contains('button', /^Launch$/).first().click();
+  log.end();
+});
+
+Cypress.Commands.add('navigateTo', (page, config = {}) => {
+  const log = Cypress.log({
+    displayName: `Navigate using to ${page}`,
+    message: [`navigate to ${page}`],
+    autoEnd: false,
+  });
+
+  cy.get('[data-test-id="navigation-menu"]').contains('a', page).click(config);
```

Comment:
use `[data-test-id=...]`

Reply:
The comment was meant to replace `data-test-id=` that was assigned to each `a` into using `data-test-id=` on the top most element. This was decided because using `data-test-id=` on the links will require each link to use an attribute that's assigned to it e.g.`data-test-id="navigate-to-Data Management"` which isn't good practice? I'm open to suggestions.